src/WebJobs.Script/Scale/HostPerformanceManager.cs

```diff
@@ -68,8 +61,7 @@ public virtual bool PerformanceCountersExceeded(Collection<string> exceededCount
         /// </summary>
         public virtual async Task<bool> IsUnderHighLoadAsync(Collection<string> exceededCounters = null, ILogger logger = null)
```

Comment:
If the `exceededCounters` param is no longer used, should it be removed?

Reply:
Yes, will remove